mautic/mautic
# TPROD-305 Twig refactoring for Api bundle

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes https://mautic.atlassian.net/browse/TPROD-305



#### Description:

Refactors PHP templates into Twig for the Api Bundle.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Click through the Api Bundle's Ui. 
3. Create, edit, delete API credentials, global search is not broken

Known issues that already exist on the 5.x and 4.4 version:
- Search and global search always return all results no matter what you search for
- Ordering always return the same order.
- The API config form (in global configuration) loses the "Mautic blue" title. Config bundle will have to be converted to Twig to fix that.
